src/main/java/teammates/ui/controller/InstructorFeedbackQuestionEditAction.java

```diff
@@ -130,51 +146,61 @@ public static String validateQuestionGiverRecipientVisibility(
     private static FeedbackQuestionAttributes extractFeedbackQuestionData(Map<String, String[]> requestParameters) {
         FeedbackQuestionAttributes newQuestion = new FeedbackQuestionAttributes();
         
-        newQuestion.setId(HttpRequestHelper.getValueFromParamMap(requestParameters, Const.ParamsNames.FEEDBACK_QUESTION_ID));
+        newQuestion.setId(HttpRequestHelper.getValueFromParamMap(requestParameters, 
+                                                                 Const.ParamsNames.FEEDBACK_QUESTION_ID));
         Assumption.assertNotNull("Null question id", newQuestion.getId());
         
-        newQuestion.courseId = HttpRequestHelper.getValueFromParamMap(requestParameters, Const.ParamsNames.COURSE_ID);
+        newQuestion.courseId = HttpRequestHelper.getValueFromParamMap(requestParameters, 
+                                                                      Const.ParamsNames.COURSE_ID);
         Assumption.assertNotNull("Null course id", newQuestion.courseId);
         
-        newQuestion.feedbackSessionName = HttpRequestHelper.getValueFromParamMap(requestParameters, Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        newQuestion.feedbackSessionName = HttpRequestHelper.getValueFromParamMap(
+                requestParameters, Const.ParamsNames.FEEDBACK_SESSION_NAME);
```

Comment:
Try breaking after the `=` . it may give a better readability (i.e. prefer higher level breaks over lower level breaks).